contrib/epee/include/net/abstract_tcp_server2.h

```diff
@@ -267,13 +268,14 @@ namespace net_utils
       bool stop_signal_sent;
     };
 
-    /// Construct a connection with the given io_service.
-    explicit connection( boost::asio::io_service& io_service,
+    /// Construct a connection with the given io_context.
+    explicit connection( boost::asio::io_context& io_context,
```

Comment:
Shouldn't we use the `io_context_t` alias here since we're changing this line anyways?

Reply:
Updated.